app/services/katello/product_content_importer.rb

```diff
@@ -35,12 +76,13 @@ def add_product_content(product, product_content_json)
     end
 
     def find_product_for_content(content_id)
-      prod = @cp_products.find do |prod_json|
-        prod_json['productContent'].any? do |product_content_json|
+      #Hash.find converts hash into ennuberable iterating over each key,value pair which we can capture based on find result
```

Comment:
:stuck_out_tongue_winking_eye: 

```suggestion
      #Hash.find converts hash into enumerable iterating over each key,value pair which we can capture based on find result
```